__tests__/app.test.js

```diff
@@ -35,3 +35,30 @@ describe("/api", () => {
         });
     });
 });
+describe("/api/articles/:article_id", () => {
+    test("GET: 200 sends an article object for the specified article ID", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .then(({ body }) => {
+            const { article } = body;
+            expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String)
+            })
+        })
+    })
+    test("GET: 404 sends an error if the article ID is not found", () => {
+        return request(app)
+        .get("/api/articles/300")
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Article Not Found");
+        })
```

Comment:
Another test for status 400 - for example /api/articles/not-an-id is needed too please